src/execute/oas3/parameter-builders.js

```diff
@@ -3,24 +3,25 @@ import { resolve as resolvePathTemplate } from 'openapi-path-templating';
 import stylize, { encodeCharacters } from './style-serializer.js';
 import serialize from './content-serializer.js';
 
-export function path({ req, value, parameter, pathName }) {
+export function path({ req, value, parameter, baseURL }) {
   const { name, style, explode, content } = parameter;
 
   if (value === undefined) return;
 
+  const pathname = req.url.replace(baseURL, '');
```

Comment:
Let's make it safe by really striping the baseURL only from the begging of the `req.url`.

```suggestion
  const pathname = req.url.replace(new RegExp(`^${escapeRegExp(baseURL)}`), '');
```

Reply:
Never mind, it's equivalent with `string` replace.